src/goto-symex/symex_main.cpp

```diff
@@ -136,8 +136,17 @@ void goto_symext::symex_entry_point(
   state.top().end_of_function=limit;
   state.top().calling_location.pc=state.top().end_of_function;
   state.symex_target=&target;
-  state.dirty=util_make_unique<dirtyt>(goto_functions);
-
+  state.dirty=util_make_unique<incremental_dirtyt>();
+  if(!pc->function.empty())
```

Comment:
This ought to be an invariant. See https://github.com/diffblue/cbmc/pull/1799 for a discussion.

Reply:
Suggesting merging https://github.com/diffblue/cbmc/pull/1843 then it will be